core/storage/topic/gae_models_test.py

```diff
@@ -263,6 +369,18 @@ def test_has_reference_to_user_id(self) -> None:
             self.assertFalse(
                 topic_models.TopicRightsModel.has_reference_to_user_id('x_id'))
 
+            topic_rights.manager_ids = []
```

Comment:
Why do you need to do this if you have the call below? It seems to me like the call below should take care of it.